components/system/components/Buttons.js

```diff
@@ -59,7 +59,58 @@ const STYLES_BUTTON_PRIMARY = css`
   }
 `;
 
+const STYLES_BUTTON_PRIMARY_FULL = css`
+  ${STYLES_BUTTON_FULL}
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+  background-color: ${Constants.system.brand};
+  color: ${Constants.system.white};
+
+  :hover {
+    background-color: #003fe3;
+  }
+
+  :focus {
+    box-shadow: inset 0 0 5px 2px rgba(0, 0, 0, 0.3);
+    background-color: ${Constants.system.brand};
+    outline: 0;
+    border: 0;
+  }
+`;
+
 export const ButtonPrimary = (props) => {
+  if (props.full) {
+    if (props.loading) {
+      return (
+        <button css={STYLES_BUTTON_PRIMARY_FULL} style={props.style}>
+          <LoaderSpinner style={{ height: 16, width: 16 }} />
+        </button>
+      );
+    }
+
+    if (props.type === "label") {
+      return (
+        <label
+          css={STYLES_BUTTON_PRIMARY_FULL}
+          style={props.style}
+          onClick={props.onClick}
+          children={props.children}
+          type={props.label}
+          htmlFor={props.htmlFor}
+        />
+      );
+    }
+
+    return (
+      <button
+        css={STYLES_BUTTON_PRIMARY_FULL}
```

Comment:
This is incorrect because its not the full option, but refer to my previous comment regardless.